### PR TITLE
Tests for Space Designer Form Component

### PR DESCRIPTION
**Pull request by adityanaag3**
### What does this PR do?
Added tests for Space Designer Form component

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

File: es-space-mgmt/main/default/lwc/spaceDesignForm/__tests__/spaceDesignForm.test.js [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+                expect(lightningLayoutEl).not.toBeNull();
+
+                const lightningLayoutItemsEl = element.shadowRoot.querySelectorAll(
+                    'lightning-layout-item'
+                );
+                expect(lightningLayoutItemsEl.length).toBe(2);
+            });
+        });
+
+        it('shows a error panel when getRelatedSpaces wire is not successfull', () => {
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/spaceDesignForm/__tests__/spaceDesignForm.test.js]: typo: "shows an error panel..."

File: es-space-mgmt/main/default/lwc/spaceDesignForm/__tests__/spaceDesignForm.test.js [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+                );
+                expect(errorPanelEl).not.toBeNull();
+                expect(errorPanelEl.errors.body).toBe(WIRE_ERROR);
+                expect(errorPanelEl.friendlyMessage).toBe(
+                    'There was an issue loading related market data.'
+                );
+            });
+        });
+
+        it('shows a list of spaces if wire is successfull', () => {
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/spaceDesignForm/__tests__/spaceDesignForm.test.js]: typo in successful

File: es-space-mgmt/main/default/lwc/spaceDesignForm/__tests__/spaceDesignForm.test.js [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+            return Promise.resolve().then(() => {
+                const buttonEl = element.shadowRoot.querySelectorAll(
+                    'lightning-button'
+                );
+                expect(buttonEl[0].disabled).toBeTruthy();
+                expect(buttonEl[1].disabled).toBeTruthy();
+            });
+        });
+
+        it('buttons are enabled when no space is selected', () => {
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/spaceDesignForm/__tests__/spaceDesignForm.test.js]: I think that you mean "when a space is slected"

**Review by pozil — CHANGES_REQUESTED** [review]
Good tests. I requested some minor changes.

**Comment by adityanaag3** [comment]
Copy paste issues :) Fixed them all

**Review by pozil — APPROVED** [review]

**Pull request merged**